tests/integration/features/misc.feature

```diff
@@ -367,3 +367,35 @@ Feature: Various utility commands.
         When I set network -> custom-headers to {"X-Qute-Test": "testvalue"}
         And I open headers
         Then the header X-Qute-Test should be set to testvalue
+
+    ## :messages
+
+    Scenario: Showing error messages
+        When I run :message-error the-error-message
+        And I run :message-warning the-warning-message
+        And I run :message-info the-info-message
+        And I run :messages
+        Then the error "the-error-message" should be shown
+        And the warning "the-warning-message" should be shown
+        And the page should contain the plaintext "the-error-message"
```

Comment:
Maybe also check the page does *not* contain the other messages, i.e. they are filtered correctly? Simply add a "the page should not contain the plaintext" step similar to `check_contents_plain` in `tests/integration/features/conftest.py`.